### DISCUSSION TITLE
Rebuild attached container, after base container is rebuilt?

### DISCUSSION THREAD

**Discussion by BrodyStone21** [containrrr/watchtower#1273] [Q&A] [answered]
Hopefully, my wording in the title isn't confusing. Let me try to explain two different container configurations that I have. 

vpn -> qbittorrent

unbound -> pihole

If I update the "vpn" container, "qbittorrent" cannot work because it's attached directly to the old container. I'll have to remove "qbittorrent", and then rebuild it, attaching it to the new container. I can update "qbittorrent" without having to rebuild "vpn". 

How can I make watchtower rebuild "qbittorrent" every time "vpn" is updated?

I do not use docker-compose; I use docker run. Is this possible? Right now I have "vpn" set with a label telling watchtower to only monitor it.

**Comment by piksel** [comment, marked as answer]
Yes, using a `depends-on` label or `--link vpn`, see https://containrrr.dev/watchtower/linked-containers/

**Reply by BrodyStone21** [reply]
Thank you sir, appreciate it.
